pkg/flowkit/config/processor.go

```diff
@@ -43,8 +43,14 @@ func ProcessorRun(raw []byte) ([]byte, map[string]string) {
 
 // processEnv finds env variables and insert env values.
 func processEnv(raw string) string {
-	_ = godotenv.Load() // try to load .env file
-
+	path, _ := os.Getwd()
+	envFilePath := path + "/private_keys.env"
+	_, error := os.Stat(envFilePath)
+	// check if error is not "file not exists"
+	if error == nil {
+		godotenv.Load(envFilePath)
+	}
+	_ = godotenv.Load()
```

Comment:
I have left this line behind because of https://docs.onflow.org/flow-cli/security/#store-configuration-in-environment-variables, but I don't think this line actually loads any env, because it reads only from the existing directory, which in this case will be config/processor.go. If we really want to read the env files, maybe it should be from cmd/flow or just in the flowcli directory itself.